src/microbe_stage/membrane_system.h

```diff
@@ -28,9 +31,18 @@ class MembraneComponent : public Leviathan::Component {
     };
 
 public:
-    MembraneComponent();
+    MembraneComponent(MEMBRANE_TYPE type);
 	virtual ~MembraneComponent();
 
+
+
+	//holder for membrane type
+	MEMBRANE_TYPE membraneType;
+
+	void setMembraneType(MEMBRANE_TYPE type);
```

Comment:
There should be a comment about this not taking affect without reseting this membrane as only that causes the mesh to be re-generated.